src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+
+.page__body {
```

Comment:
According to the checklist, avoid using '*' selector as it impacts performance. Please remove it.